packages/angular_devkit/build_angular/src/tslint/index.ts

```diff
@@ -64,7 +43,7 @@ async function _run(
     throw new Error('A "project" must be specified to enable type checking.');
   }
 
-  const projectTslint = await _loadTslint();
+  const projectTslint = await import('tslint');
```

Comment:
Can you wrap this in a try/catch with a builder error result returned if this fails to find tslint?  With it being an optional dependency it is a potential error case.

Reply:
Done